packages/loot-core/src/types/server-handlers.d.ts

```diff
@@ -28,6 +28,8 @@ export interface ServerHandlers {
 
   'transaction-delete': (transaction) => Promise<EmptyObject>;
 
+  'transaction-move': (arg: { id; accountId; targetId }) => Promise<unknown>;
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ideally we shouldn't use `unknown` and `any` types for new code we add.